doc/Language/objects.pod6

```diff
@@ -264,6 +264,54 @@ Method names can be resolved at runtime with the C<.""> operator.
     A.new."$name"().say;
     # OUTPUT: «(Any)␤»
 
+A method's signature can have an I<invocant> as its first parameter
+followed by a comma, which allows for the method to refer to the object
```

Comment:
This should be "colon", not "comma".

Reply:
Doh! 🤦‍♂️ Thanks for the catch @moritz !